### DISCUSSION TITLE
CRASH ON STARTUP!!!!

### DISCUSSION THREAD

**Discussion by Straight-Circle** [Sinytra/ForgifiedFabricAPI#72] [General]
[crash-2023-10-29_22.08.43-client.txt](https://github.com/Sinytra/ForgifiedFabricAPI/files/13200063/crash-2023-10-29_22.08.43-client.txt)

Can someone please tell me why I'm getting this error, I'm running "Forge 47.1.3" with the latest version of both Connector and Fabric as per Modrinth Post. I've searched through here and I'm just not getting it, I have tried about a million different combinations of each and none seem to work I've tried multiple different mods as well, I'm just not sure of the issue.

Edit: I have more information regarding this using ChatGPT but no means to fix the issues, further input would be appreciated.
![chatGPT error](https://github.com/Sinytra/ForgifiedFabricAPI/assets/145818180/1f6e8ee4-371d-4607-8945-64445faee17d)

**Comment by Su5eD** [comment]
Please provide your debug.log

**Reply by Straight-Circle** [reply]
so, I was originally going through the Minecraft launcher as a standalone and in the beginning after setting up a mod pack through curse forge it still wasn't working but I cleared the mods and it was working, now I've started adding mods in which its breaking again could be related to that but there was also some things related to mixin issues but I'm going to be honest I'm fairly new to modding Minecraft so I don't know what the flip I'm doing. but heres the log

[latest.log](https://github.com/Sinytra/ForgifiedFabricAPI/files/13210204/latest.log)

**Reply by Straight-Circle** [reply]
sorry I should say it wasn't working at first even only having connector and forgfab installed. i said that a little weird, this log was with additional mods though.

**Reply by Straight-Circle** [reply]
okay I've just found the compatibility page, I've tracked the issues down to optifine, sodium (realized isn't compatible), lithium as well as colormatic. do you know if there are good alternatives to these
